Scripts/Modules/DeviceInputModule.cs

```diff
@@ -37,6 +37,7 @@ sealed class DeviceInputModule : MonoBehaviour
 
 		public Action<HashSet<IProcessInput>, ConsumeControlDelegate> processInput;
 		public Action<List<ActionMapInput>>  updatePlayerHandleMaps;
+		List<InputControl> m_RemoveList = new List<InputControl>();
```

Comment:
We've been commenting these types of lists specifically with this:

// Local method use only -- created here to reduce garbage collection

Any of these lists can (and should?) be static readonly, because the idea is to clear them out before each use, and they aren't being used in methods called concurrently. This will reduce the size of the object and I think it also avoids the offset when getting the reference.

Reply:
That's why I put it in this section where the comment was -I'll make it static

Reply:
Oh hmm. This one is a little weird. The public properties aren't gc related

Reply:
I'll move above the puplic props